src/main/java/domain/Lottos.java

```diff
@@ -0,0 +1,32 @@
+package domain;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos){
+        Objects.requireNonNull(lottos);
+        this.lottos = lottos;
+    }
+
+    public LottoResult calculate(List<Integer> winningNumbers) throws Exception {
+        LottoResult lottoResult = new LottoResult();
+
+        lottos.stream().forEach(lotto -> lottoResult.addPrizeResult(lotto.checkWhetherToWin(winningNumbers)));
```

Comment:
```suggestion
        lottos.forEach(lotto -> lottoResult.addPrizeResult(lotto.checkWhetherToWin(winningNumbers)));
```
stream 을 사용하지 않아도 될거 같아요! 😄